src/hooks/useAuth.ts

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from 'react';
+import { authHookCb } from '../integrations/stytch';
+import { getStytchJwt } from '../utils/browser';
+import { getAuthClient } from '../utils/init';
+/** These next 2 should maybe be dynamically imported, but having trouble with that */
+import { useIdleTimer } from 'react-idle-timer';
+import throttle from 'lodash.throttle';
+
+const TEN_SECONDS_IN_MILLISECONDS = 1000 * 10;
+const FIVE_MINUTES_IN_MILLISECONDS = 1000 * 60 * 5;
+
+export const useAuth = ({
```

Comment:
pass in a single formName to preload (use existing preload call)

if useAuth called with multiple formNames we should raise an exception